### DISCUSSION TITLE
Task 5

### DISCUSSION THREAD

**Discussion by ckocauw** [chulminy/CIVE497-CIVE700#162] [General]
Hi Prof.,

Have you posted the Task 5?

Best regards,
Cem

**Comment by chulminy** [comment]
I will do today! Sorry about it!

**Reply by ckocauw** [reply]
You are welcome. I've asked because the deadline is close.

Best regards,
Cem

**Reply by chulminy** [reply]
Sorry. I'm working on it. I promise I will upload it before the end of today.

**Reply by chulminy** [reply]
The template code is ready but I should write their descriptions. You will see the template tomorrow.

**Reply by chulminy** [reply]
uploaded!
